th_map:
  1:
    rh_THJ1: 0.0
    rh_THJ2: 0.0
    rh_THJ3: 0.0
    rh_THJ4: 0.0
    rh_THJ5: 0.0
  0:
    rh_THJ1: 0.73
    rh_THJ2: 0.70
    rh_THJ3: 0.0
    rh_THJ4: 1.22
    rh_THJ5: 0.05
ff_map:
  1:
    rh_FFJ1: 0.0
    rh_FFJ2: 0.0
    rh_FFJ3: 0.0
    rh_FFJ4: 0.0
  0:
    rh_FFJ1: 1.5707
    rh_FFJ2: 1.5707
    rh_FFJ3: 1.5707
    rh_FFJ4: 0.0
mf_map:
  1:
    rh_MFJ1: 0.0
    rh_MFJ2: 0.0
    rh_MFJ3: 0.0
    rh_MFJ4: 0.0
  0:
    rh_MFJ1: 1.5707
    rh_MFJ2: 1.5707
    rh_MFJ3: 1.5707
    rh_MFJ4: 0.0
rf_map:
  1:
    rh_RFJ1: 0.0
    rh_RFJ2: 0.0
    rh_RFJ3: 0.0
    rh_RFJ4: 0.0
  0:
    rh_RFJ1: 1.5707
    rh_RFJ2: 1.5707
    rh_RFJ3: 1.5707
    rh_RFJ4: 0.0
lf_map:
  1:
    rh_LFJ1: 0.0
    rh_LFJ2: 0.0
    rh_LFJ3: 0.0
    rh_LFJ4: 0.0
    rh_LFJ5: 0.0
  0:
    rh_LFJ1: 1.5707
    rh_LFJ2: 1.5707
    rh_LFJ3: 1.5707
    rh_LFJ4: 0.0
    rh_LFJ5: 0.0
